src/Models/Group.php

```diff
@@ -88,6 +88,8 @@ class Group extends Model
      */
     protected $fillable = ['main_character_id'];
 
+    protected $appends = ['main_character'];
```

Comment:
Can you add the missing documentation block here ?

Reply:
done